.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,25 +1,32 @@
-### Summary
-<!-- Description of PR, with any special instructions for your reviewers. -->
 
-### Reason
-<!-- Why are you making this change? Can be a link to a Jira ticket, GH issue, 
-Trello card, etc. -->
+### Summary 
 
-### Testing
-<!-- How can your reviewers test your change? How did you test it? -->
+Description: <!-- What did you change and why? -->
+ 
+<!-- Include any supporting resources, e.g. link to a Jira ticket, GH issue, FTI, Slack, Aha, etc. -->
 
-<!-- (Optional) Include link to topic in Netlify preview after it's generated 
-(around 10mins after PR is created) -->
+Changelog entry, if applicable:
 
-<!--
 
-!!! Only Kong employees can add labels due to a GitHub limitation. If you're an OSS contributor, thank you! The maintainers will label this PR for you !!!
+### Testing instructions
 
-When raising a pull request, it's useful to indicate what type of review you're looking for from the team. To help with this, we've added three labels that can be applied:
+Netlify link: <!-- Netlify will generate a preview link after PR is opened. Add links to your edited content here. -->
+
+
+### Checklist 
+
+[ ] Review label added
+[ ] Changelog entry (if applicable)
+
+
+<!-- !!! Only Kong employees can add labels due to a GitHub limitation. If you're an OSS contributor, thank you! The maintainers will label this PR for you !!! -->
+
+<!-- When raising a pull request, indicate what type of review you need with one of the following labels:
 
     review:copyedit: Request for writer review.
     review:general: Review for general accuracy and presentation. Does the doc work? Does it output correctly?
-    review:tech: Request for technical review from an SME.
+    review:tech: Request for technical review for a platform change.
```

Comment:
Is this missing `review:sme`?

Reply:
I _think_ we had said that `review:sme` was a docs-team only label. But I don't remember the details.